parsers/verilog.c

```diff
@@ -114,6 +114,7 @@ typedef enum {
  *   DATA DEFINITIONS
  */
 static int Ungetc;
+static bool inSkipPastMatch = false;	// for vGetc() and skipPastMatch()
```

Comment:
If possible, I would like to avoid introducing a variable having a broad scope.
I frequently use the following technique to avoid introducing such a variable.

```c
static int vGetcFull (bool inSkipPastMatch)
{
   ...
  else if (inSkipPastMatch && c == '"')
  ...
}
...
static int vGetc (void)
{
    return vGetcFull (false);
}
...
static int skipPastMatch (const char *const pair)
{
   ...
   do
	{
		c = vGetcFull (true);
		if (c == begin)
			++matchLevel;
		else if (c == end)
			--matchLevel;
	}
	while (c != EOF && matchLevel > 0);
    return vGetc (); /* or vGetcFull (false); */
}
...
```

Reply:
Cool!
This technique can be used many other situations. Thanks.
I've fixed this.  (I renamed vGetcFull() to _vGetc().)